.magento.app.yaml

```diff
@@ -55,6 +55,9 @@ web:
             allow: true
             scripts: false
             passthru: "/front-static.php"
```

Comment:
That file doesn't exist, right? It's some pretty old attempt at supporting static assets in development mode. Just remove that line.